### PR TITLE
DT-186 - add endpoints for completing and failing an activity

### PR DESCRIPTION
**Pull request by rossedfort**
## What was changed

Add endpoints for Completing an Failing an activity by the activity's ID.
## Checklist

1. Closes 

2. How was this tested:

3. Any docs updates needed?

### CODE REVIEW

File: server/generated/openapi/service.swagger.json [hunk 1]
```diff
@@ -5066,7 +5214,7 @@
           "items": {
             "$ref": "#/definitions/v1CalendarSpec"
           },
-          "description": "Any timestamps matching any of exclude_* will be skipped."
+          "description": "Any timestamps matching any of exclude_* will be skipped.\n\nuse exclude_structured_calendar"
```
Comment (laurakwhit) [inline server/generated/openapi/service.swagger.json:5217]: Was this an intentional update? Only asking as it seems unrelated 😅

**Review by laurakwhit — APPROVED** [review]
Lgtm, except for those failing tests 😞

File: server/generated/openapi/service.swagger.json (hunk 1 above)
Comment (rossedfort) [reply server/generated/openapi/service.swagger.json:5217]: Hah yeah I'm not sure what this is exactly. These files are generated so I just kept what was there.
